integration-tests/common-test-code/src/main/java/com/vaadin/flow/quarkus/it/routecontext/ErrorView.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2000-20121 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.vaadin.flow.quarkus.it.routecontext;
+
+import com.vaadin.flow.router.BeforeEnterEvent;
+import com.vaadin.flow.router.BeforeEnterObserver;
+import com.vaadin.flow.router.Route;
+
+@Route("error")
+public class ErrorView extends AbstractCountedView
+        implements BeforeEnterObserver {
+
+    @Override
+    public void beforeEnter(BeforeEnterEvent event) {
+        if (true) {
```

Comment:
I'm not sure. 
This is a copy of 
https://github.com/vaadin/cdi/blob/master/vaadin-cdi-itest/src/main/java/com/vaadin/cdi/itest/routecontext/ErrorView.java
I don't know which way exactly `ErrorView` is instantiated : via CDI or via reflection.
In case it's CDI then the exception may be somehow decorated by the CDI. So may be this is a reason do not use a CTOR.
`if(true)` would be needed if there was a code below the block. So looks like this is excessive. 

But I would like to keep the code the same as it's in CDI even though it can be slightly rewritten.